lms/templates/courseware/course_about.html

```diff
@@ -319,6 +319,14 @@
             </li>
           % endif
 
+          % if pre_requisite_course:
+          <li class="prerequisite-course important-dates-item">
+            <i class="icon icon-list-ul"></i>
+            <p class="important-dates-item-title">${_("Requirements")}</p>
```

Comment:
Just thought of this: Are we listing course requirements (general skills and knowledge) and course prerequisites (specific courses the student has to pass) separately? If so, this heading should be "Prerequisites" or "Course Prerequisites".